client-side/src/components/MyButton.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import './MyButton.scss';
+
+import PropTypes from 'prop-types';
+
+import Button from '@mui/material/Button';
```

Comment:
correct order of imports is:
librarys , internal js files, CSS files from library, internal CSS files